### PR TITLE
Readme edits - Testing, testing 1,2,3...

### PR DESCRIPTION
**Pull request by Janative-zz**
@the-oem

Thoughts? 

giphy 3

### CODE REVIEW

File: README.md
Comment (the-oem) [inline README.md]: I would use a colon `:` here instead of a semi-colon `;`.

Because this is a markdown file (which you can tell by the .md file extension), you can also use all kinds of cool, fancy formatting tricks (italics, bold, lists, embedded images, etc). Here is a [great guide to learning markdown syntax](https://guides.github.com/features/mastering-markdown/) that you should take a look at.

I'll approve the PR once you add some fanciness ;)

**Review by the-oem — CHANGES_REQUESTED** [review]

File: README.md
Comment (Janative-zz) [reply README.md]: You are right about the colon. Dammit. I hate it when you're right. 

I'll work on the fancy.

Comment (Janative-zz) [reply README.md]: Notice - I did change what I could to use the markdown syntax you suggested.

**Comment by the-oem** [comment]
The changes look good! 👍 
But you are going to have to resolve your merge conflicts before you can merge it.
xoxoxo

**Pull request merged**
